spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/TraceKeys.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.instrument;
+
+/**
+ * Well-known {@link org.springframework.cloud.sleuth.Span#tag(String, String) span tag} keys.
```

Comment:
We already do this in the ruby-tracer in different ways 
HOST is set in the endpoint.
METHOD is set in the rpc_name
PATH , we call it in ruby "http.uri"
URL, we are not logging it. OK to have support but would like to make it optional as it may be some huge urls in there. 
STATUS we are logging it as "http.status"